packages/design-system-scripts/jest/unit.config.js

```diff
@@ -9,6 +9,7 @@ module.exports = (rootDir, core) => ({
   snapshotSerializers: ['enzyme-to-json/serializer'],
   testPathIgnorePatterns: ['dist/', 'node_modules/', '.+\\.e2e\\.test\\.js$'],
   transformIgnorePatterns: ['node_modules(?!/@cmsgov)'],
+  testTimeout: 30000,
```

Comment:
Is this related? Are unit tests timing out?

Reply:
Ah! This is a leftover. Updating...